Tasks/Common/packaging-common/nuget/NuGetConfigHelper2.ts

```diff
@@ -144,14 +144,16 @@ export class NuGetConfigHelper2 {
 
     private getTempNuGetConfigPath(): string {
         const tempNuGetConfigBaseDir = NuGetConfigHelper2.getTempNuGetConfigBasePath();
-        const tempNuGetConfigFileName = "tempNuGet_" + tl.getVariable("build.buildId") + ".config";
+        // Changing this to standard name of nuget.config. In one of the tasks, dotnet add command is being used and
+        // add command does not take the name of the config file as parameter. It assumes default name.
+        // This won't impact anything as this file is anyway deleted right after the operation.
+        const tempNuGetConfigFileName =  "nuget.config";
```

Comment:
This will not bode well with cases where, user specifies a nugetConfigPath which has nuget.config already present. Like in DotNetCore CLI task and might be other as well

Reply:
I will discuss this with you.

Reply:
Closing this with the updated design.